<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,1">
     <item row="3" column="0">
      <widget class="QLabel" name="label_heater_duty_cycle">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Heater max duty cycle</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="inGUI_MaxTemp">
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>130</number>
       </property>
       <property name="value">
        <number>130</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_heater_p_actie">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Heater P-actie</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_MaxTemperatuur">
       <property name="text">
        <string>Max. Temperatuur</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_StopTemperatuur">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Stop Temperatuur</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_StopTemperatuur2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>[℃]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_heater_p_actie_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>[ - ]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_CT4">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Log interval</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="inGUI_StopTemp">
       <property name="maximum">
        <number>130</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="inGUI_LogInterval">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>900</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_Temperaturen_3">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>[s]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="inGUI_heater_max_duty_cycle">
       <property name="toolTip">
        <string extracomment="Geef hier een waarde voor de temperatuur waarbij de heaters moeten worden uitgeschakeld."/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="inGUI_heater_p_actie">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_heater_duty_cycle_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>[%]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_Temperaturen_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>[℃]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>(max. 130 ℃)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>(2...900 s)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>(stop loggen als gem. temp lager is dan deze waarde)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>(1...25 %)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>(1...10 =&gt; x/10 =&gt; p = 0.1...1.0)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_OK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
